tests/Carbon/SettersTest.php

```diff
@@ -241,7 +241,7 @@ public function testTimestampSetter()
         $d->timestamp = 1600887164.88952298;
         $this->assertSame('2020-09-23 14:52:44.889523', $d->format('Y-m-d H:i:s.u'));
 
-        $d->setTimestamp(1599828571.23561248);
+        $d->setTimestamp(1599828571.235612);
```

Comment:
PHP 8.4 has a problem with rounding, but I don't think it's OK to just mute it. Users can pass more than 6 digits so it's a case we should test.

Reply:
Is it okay if I update it as follows?

``` php 
if (PHP_VERSION_ID < 80400) {
    $d->setTimestamp(1599828571.23561248);
    $this->assertSame('2020-09-11 08:49:31.235612', $d->format('Y-m-d H:i:s.u'));
}
```

Reply:
Because it's output 1599828571.235613 on PHP 8.4

Reply:
https://github.com/CarbonPHP/carbon/pull/13/commits/8910c5114f60f2a918aa2d9ae2c14056dcb71e87

Reply:
FTR I think this issue comes from the same root cause as https://github.com/php/php-src/issues/14332